static/js/vue-cdr-access/tests/unit/facetTags.spec.js

```diff
@@ -0,0 +1,156 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+import VueRouter from 'vue-router';
+import facetTags from '@/components/facetTags.vue';
+
+const localVue = createLocalVue();
+localVue.use(VueRouter);
+const router = new VueRouter({
+    routes: [
+        {
+            path: '/search/:uuid?',
+            name: 'searchRecords'
+        }
+    ]
+});
+let wrapper, facet_tags;
+
+describe('facetTags.vue', () => {
+    beforeEach(() => {
+        wrapper = shallowMount(facetTags, {
+            localVue,
+            router,
+            propsData: {
+                facetList: [
+                    {
+                        name: "PARENT_COLLECTION",
+                        values: [
+                            {
+                                count: 19,
+                                displayValue: "testCollection",
+                                limitToValue: "d77fd8c9-744b-42ab-8e20-5ad9bdf8194e",
+                                value: "d77fd8c9-744b-42ab-8e20-5ad9bdf8194e",
+                                fieldName: "PARENT_COLLECTION"
+                            },
+                            {
+                                count: 1,
+                                displayValue: "test2Collection",
+                                limitToValue: "88386d31-6931-467d-add5-1d109f335302",
+                                value: "88386d31-6931-467d-add5-1d109f335302",
+                                fieldName: "PARENT_COLLECTION"
+                            }
+                        ]
+                    },
+                    {
+                        name: "CONTENT_TYPE",
+                        values: [
+                            {
+                                count: 8,
+                                displayValue: "Image",
+                                limitToValue: "image",
+                                value: "^image,Image",
+                                fieldName: "CONTENT_TYPE"
+                            },
+                            {
+                                count: 2,
+                                displayValue: "Text",
+                                limitToValue: "text",
+                                value: "^text,Text",
+                                fieldName: "CONTENT_TYPE"
+                            },
+                            {
+                                count: 2,
+                                displayValue: "png",
+                                limitToValue: "image/png",
+                                value: "/image^png,png",
+                                fieldName: "CONTENT_TYPE"
+                            }
+                        ]
+                    }
+                ]
+            }
+        });
+
+        facet_tags = wrapper.findAll('ul li');
+    });
+
+    it("displays selected tags", async () => {
+        wrapper.vm.$router.push('/search/?collection=d77fd8c9-744b-42ab-8e20-5ad9bdf8194e');
```

Comment:
for what its worth, the urls i'm seeing in the live interface are in the form `/search?...` rather than `search/?...`